### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Nkaleth**
Accomplished tasks in this PR:

- [x] Create a new app. Give it a meaningful name.
- [x] Initialize your project with Git.
- [x] Make sure that your project has Postgres database set up.

- Build your project schema:

- [x] Create and run the necessary migration files.
- [x] Table and column names should match the ERD diagram.
- [x] note: photo for users table should be a link to an image
- [x] Foreign keys should be included.
- [x] Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- [x] All columns that are foreign keys should have a corresponding index.

#Image of Table Users 
creating_data_model_users

### CODE REVIEW

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Nkaleth 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

- [x] Please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/migrate/20230503134803_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }, index: true
+      t.integer :post_id
```
[the remark this answers is not in the document]
Comment (Nkaleth) [reply db/migrate/20230503134803_create_comments.rb:5]: @GzimAsani  thanks for the feedback, I appreciate it, but if I do not use to_table: users; how the foreign key of comments will be connected to users id? Or there is another way to do it?

[the remark this answers is not in the document]
Comment (GzimAsani) [reply db/migrate/20230503134803_create_comments.rb:5]: > @GzimAsani thanks for the feedback, I appreciate it, but if I do not use to_table: users; how the foreign key of comments will be connected to users id? Or there is another way to do it?

When you use t.references :author without specifying the to_table option, Rails will automatically infer the foreign key reference based on the name of the symbol passed to the references method (:author in this case).

Again this is totally optional, you do not feel comfortable you can continue the way you started, there is nothing wrong with that.

[the remark this answers is not in the document]
Comment (Nkaleth) [reply db/migrate/20230503134803_create_comments.rb:5]: Thank you for you answer, I appreciate it.

[the remark this answers is not in the document]
Comment (GzimAsani) [reply db/migrate/20230503134803_create_comments.rb:5]: I am glad I could help

**Pull request merged**